src/plugins/index_pattern_field_editor/public/open_editor.tsx

```diff
@@ -92,7 +92,7 @@ export const getFieldEditorOpener = ({
 
     if (fieldName && !field) {
       const err = i18n.translate('indexPatternFieldEditor.noSuchFieldName', {
-        defaultMessage: "Field named '{fieldName}' not found on index pattern",
+        defaultMessage: "Field named '{fieldName}' not found on data view",
```

Comment:
For consistency with the field list component, I would suggest:
```suggestion
        defaultMessage: "Field named '{fieldName}' not found in this data view",
```